services/src/main/scala/cromwell/pipeline/service/GitLabProjectVersioning.scala

```diff
@@ -0,0 +1,56 @@
+package cromwell.pipeline.service
+import java.nio.file.Path
+
+import com.typesafe.config.{ Config, ConfigFactory }
+import cromwell.pipeline.datastorage.dto.{ Project, ProjectFile, Version }
+
+import scala.concurrent.{ ExecutionContext, Future }
+
+class GitLabProjectVersioning(httpClient: HttpClient)(
+  implicit executionContext: ExecutionContext
+) extends ProjectVersioning[VersioningException] {
+  import GitLabConfig._
```

Comment:
Why do you put the import into the class?

Reply:
Because this class is in the same file. And this Import is actually only for this class.